readme.md

```diff
@@ -230,6 +281,6 @@ Convert the following ActiveRecord sequence to Mongoose:
 ```
 
 ```js
-// Your answer...
+andy = db.intructor.find({name: "Andy"});
+andy.wishlist_items.insert({description: "Resin Laying Deer Figurine, Gold"});
 ```
```

Comment:
:+1:

How would you write this assuming we have access to an `Instructor` class?